aips/aip-order-book.md

```diff
@@ -0,0 +1,326 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
```

Comment:
just take the next number available so that it is globally referenceable and we don't end up with a dead link